NinchatSDKSwift/Implementations/Helpers/NINLowLevelClientProps+Extension.swift

```diff
@@ -50,16 +50,16 @@ extension NINLowLevelClientProps {
 
     static func initiate(credentials: NINSessionCredentials) -> NINLowLevelClientProps {
         let props = NINLowLevelClientProps()
-        props?.set(value: credentials.userID, forKey: "user_id")
-        props?.set(value: credentials.userAuth, forKey: "user_auth")
+        props?.set(value: credentials.userID, forKey: "user_id", ofType: String.self)
+        props?.set(value: credentials.userAuth, forKey: "user_auth", ofType: String.self)
 
         return props!
     }
 
-    public static func initiate(dictionary: [String:AnyHashable]) -> NINLowLevelClientProps {
+    public static func initiate(metadata: [String:String]) -> NINLowLevelClientProps {
```

Comment:
Ninchat API action parameters (including metadata) can include non-string values. Does this change mean that it's no longer possible to specify arbitrary values, or is that possible via some other function?

Reply:
Yes, but this is just an initializer that is called when the host application sets metadata during session initialization. Do we need to support other types (int, float, etc.) in this case too?

Reply:
Yes. Metadata can contain any types, including nested objects (string-to-any mappings).

Reply:
I'll fix it then.